src/Plot/Scale.elm

```diff
@@ -0,0 +1,32 @@
+module Plot.Scale where
+
+import Private.Scale.Linear as LinearScale
+import Private.Scale.OrdinalBands as OrdinalBands
+import Private.Extras.Set as Set exposing (Set)
+import Private.Scale exposing (Scale)
+
+type alias LinearScale = Scale Set Float
+type alias OrdinalScale = Scale (List String) String
+
+linear : (Float, Float) -> (Float, Float) -> Int -> LinearScale
+linear domain range numTicks =
+  { domain = Set.createFromTuple domain
+  , range = Set.createFromTuple range
+  , interpolate = LinearScale.interpolate
+  , uninterpolate = LinearScale.uninterpolate
```

Comment:
I'm not convinced that `interpolate` and `uninterpolate` are the correct names here. Really what they are doing is mapping from to domain into the range and vice versa, respectively. I think `interpolate` is a pretty good name, but I think `uninterpolate` is bad enough to warrant renaming them both. Unless this is terminology used elsewhere.